src/configs/react.ts

```diff
@@ -91,7 +99,27 @@ export async function react(
         // react refresh
         'react-refresh/only-export-components': [
           'warn',
-          { allowConstantExport: isAllowConstantExport },
+          {
+            allowConstantExport: isAllowConstantExport,
+            allowExportNames: isUsingNext
+              ? [
+                  'config',
+                  'generateStaticParams',
+                  'metadata',
+                  'generateMetadata',
+                  'viewport',
+                  'generateViewport',
+                ]
+              : isUsingRemix
+                ? [
+                    'meta',
+                    'links',
+                    'headers',
+                    'loader',
+                    'action',
+                  ]
+                : undefined,
+          },
```

Comment:
I would allow all of them directly without checking the packages - would that have any bad side-effects?

Reply:
I don't think it should be a big problem, but when using pure vite and react, if the exported name coincides with the preset, it will still affect fast refresh.